techniques/flash/FLASH13.html

```diff
@@ -12,7 +12,7 @@ <h3>Using the language of the page as whole in the embedded Flash</h3>
          
             <p>This example defined the content of the entire web page to be in the   French language. The Flash content will inherit the specified language   from the HTML container. </p>
          
-         <pre xml:space="preserve">&lt;?xml version="1.0" encoding="UTF-8"?&gt;
+         <pre>&lt;?xml version="1.0" encoding="UTF-8"?&gt;
```

Comment:
```suggestion
         <pre><code>&lt;?xml version="1.0" encoding="UTF-8"?&gt;
```

and matching closing `</code>` in https://github.com/w3c/wcag/pull/4611/files#diff-658467b219bc5b9acf64f98624b36b4976461f2b96b87fdb978f51c7e1275e05R40